package.json

```diff
@@ -1,10 +1,14 @@
 {
-  "name": "since-time-ago",
-  "version": "1.1.1",
-  "description": "npm module to convert time stamp into a readable format",
+  "name": "since-time-ago-typescript",
+  "version": "2.1.1",
+  "description": "typescript package to convert time stamp into a readable format",
   "license": "MIT",
+  "scripts": {
+    "build": "tsc --declaration",
+    "test": "npm run build && jest test.js"
+  },
   "repository": {
-    "url": "https://github.com/Ihapmustapha/since-time-ago"
+    "url": "https://github.com/mrbrianevans/since-time-ago"
```

Comment:
For some reason, I didn't notice your merge request until now (probably because I was moving to a new country back then and was busy doing other things). Thanks for the effort, your changes make sense to me but I'd ask to keep the original repo URL and republish the package from a pipeline (GitHub action) that we can create here instead if you wouldn't mind. I see also that you did publish the package again under the name of "since-time-ago-typescript" so maybe we can add you to this repo as a contributor, archive your fork and use the new npm registery to push the upcoming releases/artifacts. Thanks again and sorry for the absence. Let me know wdyt.

Reply:
Yes, seems like a good plan! Lets revert the last 3 commits (back to v2.1.0) and merge the PR, so that typescript definitions get published in your original package.